examples/stable-diffusion/text_to_video_generation.py

```diff
@@ -178,38 +185,61 @@ def main():
         kwargs["torch_dtype"] = torch.float32
 
     # Generate images
-    pipeline: GaudiTextToVideoSDPipeline = GaudiTextToVideoSDPipeline.from_pretrained(
-        args.model_name_or_path, **kwargs
-    )
-    set_seed(args.seed)
-    outputs = pipeline(
-        prompt=args.prompts,
-        num_videos_per_prompt=args.num_videos_per_prompt,
-        batch_size=args.batch_size,
-        num_inference_steps=args.num_inference_steps,
-        guidance_scale=args.guidance_scale,
-        negative_prompt=args.negative_prompts,
-        eta=args.eta,
-        output_type="pil" if args.output_type == "mp4" else args.output_type,  # Naming inconsistency in base class
-        **kwargs_call,
-    )
-
-    # Save the pipeline in the specified directory if not None
-    if args.pipeline_save_dir is not None:
-        pipeline.save_pretrained(args.pipeline_save_dir)
-
-    # Save images in the specified directory if not None and if they are in PIL format
-    if args.video_save_dir is not None:
-        if args.output_type == "mp4":
-            video_save_dir = Path(args.video_save_dir)
-            video_save_dir.mkdir(parents=True, exist_ok=True)
-            logger.info(f"Saving images in {video_save_dir.resolve()}...")
-
-            for i, video in enumerate(outputs.videos):
-                filename = video_save_dir / f"video_{i + 1}.mp4"
-                export_to_video(video, str(filename.resolve()))
-        else:
-            logger.warning("--output_type should be equal to 'mp4' to save images in --video_save_dir.")
+    if args.pipeline_type[0] == "sdp":
```

Comment:
```suggestion
    if args.pipeline_type == "stable_diffusion":
```
No need for [0] if we don't use nargs anymore

Reply:
resolved